docs/openid.md

```diff
@@ -446,3 +446,51 @@ The synapse config will look like this:
       config:
         email_template: "{{ user.email }}"
 ```
+
+## Django OAuth Toolkit
+
+[django-oauth-toolkit](https://github.com/jazzband/django-oauth-toolkit) is a
+Django application providing out of the box all the endpoints, data and logic
+needed to add OAuth2 capabilities to your Django projects. It supports
+[OpenID Connect too](https://django-oauth-toolkit.readthedocs.io/en/latest/oidc.html).
+
+**Configuration on Django's side:**
```

Comment:
You've ended up with the synapse config under this heading. I suggest you unbold it:

```suggestion
Configuration on Django's side:
```